test/e2e_new/broker_conformance_test.go

```diff
@@ -36,6 +40,14 @@ import (
 )
 
 func TestBrokerConformance(t *testing.T) {
+	if broker.EnvCfg.BrokerClass == kafka.NamespacedBrokerClass {
+		// TODO
+		// TODO
+		// TODO
+		// TODO
+		t.Skip("Skipping conformance test for namespaced broker")
+	}
+
```

Comment:
There are limitations in broker conformance tests upstream. This part needs a little work, possibly in the upstream to make the test cases extensible/decoratable.

I still need to figure this part out.